### PR TITLE
fix(docs): default account info docs changes

### PR DESCRIPTION
**Pull request by adityadeshlahre**
## What does this PR do?
 Minor docs changes about default account !

Fixes # (issue)
image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [ ] Ran `pnpm build`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues
- [ ] First PR at Formbricks? [Please sign the CLA!](https://formbricks.com/clmyhzfrymr4ko00hycsg1tvx) Without it we wont be able to merge it 🙏

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/formbricks-com/app/docs/contributing/setup/page.mdx [hunk 1]
```diff
@@ -90,6 +90,8 @@ To get the project running locally on your machine you need to have the followin
 
    **You can now access the Formbricks app on [http://localhost:3000](http://localhost:3000)**. You will be automatically redirected to the login. To use your local installation of formbricks, create a new account.
 
+   <Note>There are no default values or default accounts in general. You can just create a new account and proceed.</Note>
```
Comment (ShubhamPalriwala) [inline apps/formbricks-com/app/docs/contributing/setup/page.mdx:93]: Can you change this to
`A fresh setup does not have a default account. Please create a new account and proceed accordingly.`

**Review by ShubhamPalriwala — CHANGES_REQUESTED** [review]
Can you rephrase it? And then it should be gtg

**Comment by adityadeshlahre** [comment]
image

updated statement looks like !

**Review by ShubhamPalriwala — APPROVED** [review]
LGTM! Thanks 🚀
